tools/dgeni/templates/property.template.html

```diff
@@ -25,7 +25,11 @@
     <p class="docs-api-property-name">
       {$ property.name $}
     </p>
-    <code class="docs-api-property-type">{$ property.type $}</code>
   </td>
-  <td class="docs-api-property-description">{$ property.description | marked | safe $}</td>
+  <td class="docs-api-property-description">
+    <div class="docs-api-property-type">
+      Type: <code>{$ property.type $}</code>
```

Comment:
I'm not a huge fan of having "Type: " explicitly. Repeated text like this creates a lot of visual noise

Reply:
Gotcha. Should I just remove type and leave it like is otherwise.

Reply:
Can you post a SS of what that looks like?